app/assets/stylesheets/map.css

```diff
@@ -0,0 +1,22 @@
+#map {
+    height: 100%;
+  }
+  
+
+  #test_div{
+
+    width: 300px;
+    height: 300px;
+    background-color:blue;
+
+  }
+  
+  #test_p{
+    font-size:50px;
+
+  }
+  html, body {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+  }
```

Comment:
The indentation seems off in this file. Also, make sure the spacing is consistent. The html, body chunk is a good example of what the rest of the file should look like. Also, remember to add a newline at the end of the file.

Sorry, I'm a bit nitpicky about formatting.

Reply:
Fixed